pages/about.md

```diff
@@ -8,35 +8,70 @@ permalink: /about/
+
+When it is safe to gather again, we plan to rent a new space. We intend to reopen in San Francisco or Oakland near a BART station, maintaining our commitment to ADA-compliant (or better) accessibility and a warm and welcoming environment.
+
+#### Growth in membership gender criteria
+
+Double Union began in 2013 with the membership gender criteria of people who “identify as a woman in a way significant to them”. In January 2018, members decided to expand this criteria to “identify as a woman or nonbinary person in a way significant to them”. In April 2019, we simplified this phrasing to “all women and nonbinary people” and expanded the description of this on [the membership page]({{ 'membership' | absolute_url }}).
+
+### Decision-making
 
-Double Union’s decision-making structure is heavily influenced by Jo Freeman’s The Tyranny of Structurelessness and has evolved over time as the organization has grown. At first, the people meeting informally made a lot of the founding decisions. After a couple of months, three people volunteered to incorporate Double Union and became the board of directors, responsible for making legal, financial, and high-level decisions. They also created the position of voting member, to decide on new member applications and take on higher-level responsibilities within the organization. A few weeks later, they created committees to make decisions about specific areas like screen printing and electronics. After about a year, Double Union members created committees to formalize and document all of our official and de-facto decision-making policies. We also paid multiculturalism and inclusion consultants about 15% of our annual budget to advise us on making Double Union more welcoming to a wider variety of women and nonbinary people.
+Double Union’s decision-making structure is heavily influenced by Jo Freeman’s essay ["The Tyranny of Structurelessness"](https://www.jofreeman.com/joreen/tyranny.htm), and it has evolved over time as the organization has grown.
 
+#### Board members
 
-### Decisionmaking at Double Union
+At first, the people meeting informally made a lot of the founding decisions. After a couple of months, three people volunteered to incorporate Double Union and became the board of directors, responsible for making legal, financial, and high-level decisions.
 
-We have a members-only mailing list and internal chat system where we propose and discuss things, and we have members meetings about once a month. This usually results in decisions without a formal process being necessary. Groups of 2+ members interested in a topic form committees, to run events or be in charge of some aspect of the space's operation.
+Each board member is limited to a two-year term, so our board membership has cycled many times since our founding in 2013. This reduces burnout, ensures that this type of power rotates among members, and encourages a wide range of perspectives on the board over time.
```

Comment:
We changed the terms in ~2018 to be 2 years plus an optional additional 3rd to help transition new members onto the board.

Reply:
@Imperiopolis Thank you, updated that part!